### PR TITLE
Tirth patel

### PR DESCRIPTION
**Pull request by tirth-patel01**
learn Panda and work with data and data exploration.

### CODE REVIEW

File: src/alco_corr_cardio.py [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
 
 print(df.head(5))
 print(df.info())
 print(df.isnull().head(5))
-# Alcohol intake | subjective Feature | alco | binary (0, 1)
 print(df["alco"].isnull())
 print(df["alco"].isna().all())
 print(df["alco"].isin([0,1]).all())
+
+from matplotlib import pyplot as plt
```
Comment (prakash-pun) [inline src/alco_corr_cardio.py:16]: move this import to top

**Review by prakash-pun — CHANGES_REQUESTED** [review]

File: src/alco_corr_cardio.py (hunk 1 above)
Comment (tirth-patel01) [reply src/alco_corr_cardio.py]: ok done.

Comment (tirth-patel01) [reply src/alco_corr_cardio.py]: please check it and merge

**Review by amannain122 — APPROVED** [review]
Please co relate with target variable and try scaling.

**Pull request merged**